addon/utils/pretty-percentage.js

```diff
@@ -0,0 +1,10 @@
+export default function prettyPercentage(decimalValue, precision=0) {
```

Comment:
Hm. That's a good question. I reasoned that unit conversion included converting from decimal to percentage, but I can see an argument for that not being the case. If it means creating another package, I guess I could do that.